recipes/torch-pme/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "torch_pme" %}
```

This is the name and should be the same as PyPI.

```suggestion
{% set name = "torch-pme" %}
```